cluster/examples/kubernetes/ceph/direct-mount.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-tools
```

Comment:
As the toolbox still exists, it would be good to rename it to, e.g., `rook-direct-mount`, that there are no name conflicts.

Reply:
Yes, that makes sense :)